templates/openstack/architecture.html

```diff
@@ -0,0 +1,280 @@
+                  OpenStack enables on-demand resource provisioning. This means that users can request them at any time through the client, dashboard or APIs. No admin intervention is required. No ticket is needed to process users' requests. Provisioning happens behind the scenes. OpenStack takes care of everything.
+                </p>
+              </div>
+            </div>
+          </li>
+          <hr class="p-rule--muted u-hide--large" />
+          <li class="p-stepped-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <h3 class="p-heading--5 p-stepped-list__title">OpenStack handles the request</h3>
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  Depending on the type of a request, one or another service gets involved in its processing. Each service manages its own domain and provides all necessary functions to handle the given task. This includes user authentication, authorization, resource scheduling, metrics collection, etc.
+                </p>
+              </div>
+            </div>
+          </li>
+          <hr class="p-rule--muted u-hide--large" />
+          <li class="p-stepped-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <h3 class="p-heading--5 p-stepped-list__title">OpenStack provisions resources</h3>
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  In order to provision the actual resource (a VM, a block storage volume, a snapshot or an external IP address), OpenStack relies on other technologies. This includes the <a href="https://ubuntu.com/server/docs/virtualisation-with-qemu">KVM</a> hypervisor, <a href="https://ubuntu.com/blog/data-centre-networking-what-is-ovn">OVN</a> for networking, <a href="https://ubuntu.com/ceph">Ceph</a> for storage and many, many more.
+                </p>
+              </div>
+            </div>
+          </li>
+        </ol>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>
+          Get started
+          <br />
+          with Canonical OpenStack
+        </h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>Are you ready for the next step? Get your own OpenStack instance up and running in just a few minutes.</p>
+          <p>
+            With Canonical OpenStack you can get started on your journey on a single physical machine. Simply try it out and continue learning through some practical exercises.
+          </p>
+        </div>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive" href="https://microstack.run/docs/single-node">Start today</a>
```

Comment:
Same here,  just a heads up that it would be `/microstack/docs/docs/<title>`, emphasis on double /docs

Reply:
Same reason here. The double /docs in URL will be resolved through Discourse, but that's not a blocker for this PR